status/templatetags/status_filters.py

```diff
@@ -121,10 +132,7 @@ def show_detail(submission, user):
     contests = get_running_contests()
```

Comment:
Problem owner can view submission details in contest duration. I'm just wondering if it's possible to follow the same rule for contest.